extensions/amp-link-rewriter/0.1/link-rewriter.js

```diff
@@ -103,6 +102,19 @@ export class LinkRewriter {
     );
   }
 
+  /**
+   * @param {!Element} anchor
+   * @return {boolean}
+   * @private
+   */
+  isNotFiltered_(anchor) {
+    if (!this.configOpts_.scopeDocument) {
+      return isElementOnScope(anchor, this.configOpts_);
```

Comment:
nit: I'd prefer `isElementInScope`

Reply:
instead of isElementOnScope?

Reply:
Yes, I think `In` makes more sense conceptually.

Reply:
Got it!